src/module/Ui/templates/entity/view/partials/text-exercise.twig

```diff
@@ -9,23 +9,29 @@
 {% set feedbackContent = markdown().toHtml(singleChoice().fetchPositiveFeedback(entity)) %}
 {% set positiveFeedback = feedbackContent ? feedbackContent : defaultFeedback %}
 {% set inputChallenge = inputChallenge().fetchInput(entity) %}
-<article class="text-exercise" itemscope itemtype="http://schema.org/Article">
+<article class="text-exercise  {{ multipleChoices|length>4 or singleChoices|length>4 ? 'extended' }}" itemscope itemtype="http://schema.org/Article">
     <section class="row">
         <div class="col-xs-10 col-sm-8 col-md-8" itemprop="articleBody">
             {{ markdown().toHtml(revision.get('content')) | raw }}
         </div>
         <div class="col-xs-2 col-sm-4 col-md-4">
             <div class="pull-right">
                 <div class="btn-group">
-                    {% set subitems = singleChoices|length>0 ? singleChoices : (multipleChoices|length>0 ? multipleChoices : []) %}
+                    {% set subitems = [] %}
+                    {% for answer in singleChoices %}
+                        {% set subitems = subitems|merge([answer['entity']]) %}
```

Comment:
It is currently only used here. I added this in #346, so there are no other usecases yet.